src/main/java/seedu/task/logic/commands/UndoCommand.java

```diff
@@ -0,0 +1,29 @@
+package seedu.task.logic.commands;
+
+import java.util.logging.Logger;
+import seedu.task.commons.exceptions.UndoableException;
+import seedu.task.logic.LogicManager;
+import seedu.taskcommons.core.LogsCenter;
+
+
+/**
+ * Undoes the most recent modification to the TaskBook
+ * @author xuchen
+ *
+ */
+public class UndoCommand extends Command{
+	private final Logger logger = LogsCenter.getLogger(UndoCommand.class);
+	
+	public static final String MESSAGE_UNDO_FAILURE = "There is no more operation can be undone.";
```

Comment:
Minor grammar mistake. Can change this to "No more operations to undo" or "There are no more operations which can be undone. "